README.md

```diff
@@ -56,7 +56,3 @@ Remove a WebSocket mask on the given data.;w
 ```js
 bu.unmask(buffer, mask);
 ```
-
-## License
-
-MIT
```

Comment:
Instead or removing it from the README I would add a link to the LICENSE file (`[MIT](LICENSE)`).